core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -335,7 +361,7 @@ class DynamicBrokerReconfigurationTest extends QuorumTestHarness with SaslSetup
   @Test
   def testKeyStoreAlter(): Unit = {
     val topic2 = "testtopic2"
-    TestUtils.createTopic(zkClient, topic2, numPartitions, replicationFactor = numServers, servers)
+    TestUtils.createTopicWithAdmin(adminClients.head, topic2, servers, numPartitions, replicationFactor = numServers)
```

Comment:
this should be `brokers`, not `servers`, right?

Also, can you add a comment referencing the JIRA for converting this test (I think you said you made one for this)

Also for any other unconverted ones I guess